### DISCUSSION TITLE
Raspberry Pi 4 feature coverage

### DISCUSSION THREAD

**Discussion by KiwiGrenade** [rsta2/circle#456] [Q&A] [answered]
Hi all!

I want to ask about feature coverage of previous Raspberry Pi editions, particularly RPi 4. I see that on the main page there is a table for RPi5 feature coverage, but as far as I can tell there is no such thing for previous RPi versions. Does that mean that everything feature mentioned in RPi 5 document is covered in all previous versions?

**Comment by rsta2** [comment, marked as answer]
Every feature in the feature table in the main README.md file is supported on 
the Raspberry Pi 4, if not noted otherwise in the left column in this table.

The Raspberry Pi 5 has a special column in this table, because the features 
differed very much for this model, when Circle 46 was released. In the meantime 
there are not that many differences any more on the develop branch.

**Reply by KiwiGrenade** [reply]
Thank you very much for quick reply!
